errorfinder/errorfinder.go

```diff
@@ -0,0 +1,80 @@
+package errorfinder
+
+import (
+	"bufio"
+	"strings"
+)
+
+// FindXcodebuildErrors ...
```

Comment:
I'm not a fan of documenting every public method, but in this case, it would be great to quickly see an example input and the returned string list.

Reply:
I'm okay with that, thanks for explaining the reasoning. (After writing my comment, I immediately noticed the test cases, those also serve as a kind of documentation :) )